services/web/client/source/class/osparc/component/metadata/ServicesInStudyUpdate.js

```diff
@@ -100,16 +100,28 @@ qx.Class.define("osparc.component.metadata.ServicesInStudyUpdate", {
           osparc.component.message.FlashMessenger.logAs(this.tr("Some service information could not be retrieved"), "WARNING");
           break;
         }
+        const metadata = osparc.utils.Services.getMetaData(node["key"], node["version"]);
+        const isDeprecated = osparc.utils.Services.isDeprecated(metadata);
```

Comment:
So the backend passes you the info that this service is marked for deprecation.
Normally, deprecation notes suggest an alternative as "please use vX.Y.Z as a replacement" etc as well as the reason for deprecation (e.g. "Due to security reason, this service has been marked for deprecation").

Reply:
For now, the only information the frontend gets is when the service was deprecated or NULL